src/main/java/domain/RandomMoveStrategy.java

```diff
@@ -0,0 +1,19 @@
+package domain;
+
+import java.util.function.IntSupplier;
+
+public class RandomMoveStrategy implements MoveStrategy {
+
+    public static final int MIN_MOVABLE_DIGIT = 4;
```

Comment:
```suggestion
    private static final int MIN_MOVABLE_DIGIT = 4;
```
외부에서 사용하지 않는 상수는 막아주는 것이 좋습니다 :)